FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Dmitry Tsurkan on 24.05.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImageMapper {
```

Comment:
The `FeedImageMapper` can be marked as `final` since it's not intended to be subclassed.

```suggestion
final class FeedImageMapper {
```